docs/src/tutorial/simple_system.md

```diff
@@ -138,6 +182,24 @@ If you are not sure whether you have the latest version, please upgrade to ensur
     release energy into the *electricity\_node*.
 ![image](../figs_simple_system/simple_system_add_unit__to_node_relationships.png)
 
+-   Right click on the *unit\_\_node\_\_node* class, and select
+    *Add relationships* from the context menu. The *Add relationships*
+    dialog will pop up.
+
+-   For each of the units enter the unit under *unit*,
+    *electricity_node* under the first node and *fuel_node* under the 
+    second node. These relationships will define the relation (or behavior)
+     between the output and input of the unit.
+![image](../figs_simple_system/simple_system_add_unit__node__node.png)
+
+!!! note
+    The *unit\_\_node\_\_node* relationship is necessary to bind the flow 
+    (flows are unbound by default) and to define an efficiency.
+    The order of the nodes is important for that definition (see later on).
+    It may seem unintuitive to define an efficiency through a three-way 
+    relationship instead of a property of a unit, but this approach allows 
+    to define efficiencies between any flow coming in and out of the unit (e.g. CHP). 
```

Comment:
```suggestion
    relationship instead of a property of a unit, but this approach allows you
    to define efficiencies between any flow(s) coming in and out of the unit (e.g. CHP). 
```